Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -155,51 +156,121 @@ def add_user_to_departing_employee(self, username, departure_date=None, note=Non
+            self._sdk.detectionlists.departing_employee.remove(user_id)
         except Exception:
             return None
-        return res["alerts"]
+        return user_id
 
-    def get_alert_details(self, alert_id):
+    def get_all_departing_employees(self):
         try:
-            res = self._sdk.alerts.get_details(alert_id)
+            res = []
+            pages = self._sdk.detectionlists.departing_employee.get_all()
+            for page in pages:
+                employees = page["items"]
+                res.extend(employees)
         except Exception:
             return None
-        return res["alerts"][0]
+        return res
 
-    def get_current_user(self):
+    def add_user_to_high_risk_employee(self, username, note=None):
         try:
-            res = self._sdk.users.get_current()
+            user_id = self.get_user_id(username)
+            self._sdk.detectionlists.high_risk_employee.add(user_id)
+            if note:
+                self._sdk.detectionlists.update_user_notes(user_id, note)
         except Exception:
             return None
-        return res
+        return user_id
 
-    def remove_user_from_departing_employee(self, username):
+    def remove_user_from_high_risk_employee(self, username):
         try:
             user_id = self.get_user_id(username)
-            self._sdk.detectionlists.departing_employee.remove(user_id)
+            self._sdk.detectionlists.high_risk_employee.remove(user_id)
         except Exception:
             return None
         return user_id
 
+    def add_user_risk_tags(self, username, risk_tags):
+        try:
+            user_id = self.get_user_id(username)
+            self._sdk.detectionlists.add_user_risk_tags(user_id, risk_tags)
+        except Exception:
+            return None
+        return user_id
+
+    def remove_user_risk_tags(self, username, risk_tags):
+        try:
+            user_id = self.get_user_id(username)
+            self._sdk.detectionlists.remove_user_risk_tags(user_id, risk_tags)
+        except Exception:
+            return None
+        return user_id
+
+    def get_all_high_risk_employees(self, risk_tags=None):
+        if isinstance(risk_tags, str):
+            risk_tags = [risk_tags]
+        try:
+            res = []
+            pages = self._sdk.detectionlists.high_risk_employee.get_all()
+            for page in pages:
+                res.extend(self._get_all_high_risk_employees_from_page(page, risk_tags))
+        except Exception:
+            return None
+        return res
+
+    def _get_all_high_risk_employees_from_page(self, page, risk_tags):
+        res = []
+        for employee in page["items"]:
+            if not risk_tags:
+                res.append(employee)
+                continue
+
+            employee_tags = employee.get("riskFactors")
+            # If the employee risk tags contain all the given risk tags
+            if employee_tags and set(risk_tags) <= set(employee_tags):
```

Comment:
would we only want employees with _all_ the given risk tags to be returned, or would it make more sense to return employees with _any_ of the give risk tags?

Reply:
I'd vote _all_, it seems most intuitive.. "Give me employees with these risk factors" etc. That is also what Tony put in the story, but we can ask him what his thoughts are.